wdio.conf.js

```diff
@@ -1,4 +1,4 @@
-const allure = require('allure-commandline')
+//const allure = require('allure-commandline')
```

Comment:
Some bugs could be created if you comment this line. Why did you comment it?

Reply:
really, i forgot, it was a moment solution for something but i can change it back

Reply:
![Captura de pantalla 2023-07-13 040013](https://github.com/BaSP-m2023/juvi-megarocket-app/assets/127536596/f3207255-e60d-4def-a7f2-6b335e566e84)

Reply:
I'm having troubles with the allure, the rest it's changed, tomorrow i will ask for help so can solve and merge